### ISSUE TITLE
Pylette's Color Extraction Capabilities and OpenCV Integration

### ISSUE THREAD

**Issue by VitorCMatias** [qTipTip/Pylette#30]
I would like to know if :

- It is possible to use Pylette to extract colors from a specific region of an image instead of the entire image. 
- I would also like to know if there is any way to use an image read by OpenCV with:
`img = cv2.imread('fig.png')`

**Comment by qTipTip** [comment]
Hello, @VitorCMatias! :) 

Thanks for dropping by!

Currently, it's only possible to extract colors from the whole image. Pylette, at the moment, supports reading images from `bytes`, a file, or from a URL, but it shouldn't be too hard to implement extraction from a numpy array. 

I assume you are wrangling images mostly as `cv2`-objects? Are those numpy arrays?

**Comment by qTipTip** [comment]
I've released Pylette 2.3.0, which now has the option of passing an `image_array` as an argument to the `extract_color`-method.

Hopefully this will solve your use-case, as you can now:

```python
import cv2
import Pylette
# 1. Extract the image using `cv2.imread`: 
image = cv2.imread('path/to/image')
# 2. Manually crop the image to select the subregion you are interested in:
cropped_image = image[x_start:x_stop, y_start: y_stop]
# 3. Pass the cropped image to the extract_color function:
palette = Pylette.extract_colors(image_array=cropped_image, ...)
```

I hope this helps! Let me know if it doesn't!
